### PR TITLE
Project code review

### PR DESCRIPTION
**Pull request by Mikelobi**
**Project Code Review**

**In this project I:**

- Added files from the admission trial project to this repository.
-  Created a pull request.
-  Reviewed the project.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                <li><strong>1984</strong> - becomes a distinguished professor at Texas A&M University</li>
+                <li><strong>2005</strong> - states "we will have to double the world food supply by 2050." Argues that genetically modified crops are the only way we can meet the demand, as we run out of arable land. Says that GM crops are not inherently dangerous because "we've been genetically modifying plants and animals for a long time. Long before we called it science, people were selecting the best breeds."</li>
+                <li><strong>2009</strong> - dies at the age of 95.<br><br><br>
+                    <p><em>"Borlaug's life and achievement are testimony to the far-reaching contribution that one man's towering intellect, persistence and scientific vision can make to human peace and progress."</em></p><br>
+                   <em> -- Indian Prime Minister Manmohan Singh</em>
+                </li>
+            </ul>
+       </div>
+       <div class="footer">If you have time, you should read more about this incredible human being on his <a href="#" id="tribute-link" target="_blank">Wikipedia entry.</a></div>
```
Comment (Mikelobi) [inline index.html:44]: - [ ] We can use `footer` tag to replace `div` tag. This will make a good HTML semantic practice 👍

File: index.html [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    <div class="container">
+        <div class="head-content">
+            <h1 id="Dr. Norman Borlaug">Dr. Norman Borlaug</h1>
+            <p>The man who saved a billion lives</p>
+            <figure id="img-div">
+                <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+                <figcaption id="img-caption" >Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</figcaption>
+            </figure> 
+        </div>
+        <div class="container-list" id="tribute-info">
```
Comment (Mikelobi) [inline index.html:20]: - [ ] Lets make sure that this is a `section` tag for better HTML semantic. I suggest `section` should be the parent tag. Please lets replace `div` tag with `section` tag 👍 .

File: index.html [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="tribute.css">
+    <title>Dr. Roman Borlaug</title>
+</head>
+<body id="main">
+    <div class="container">
+        <div class="head-content">
```
Comment (Mikelobi) [inline index.html:12]: - [ ] Please it will be nice to apply HTML semantic best practice by replacing `div` with `header`  👍.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="tribute.css">
+    <title>Dr. Roman Borlaug</title>
+</head>
+<body id="main">
+    <div class="container">
```
Comment (Mikelobi) [inline index.html:11]: - [ ] Lets consider replacing the generic `div` tag with a HTML5 semantic tags such as `main` to wrap the main content 🤤

**Review by Mikelobi — COMMENTED** [review]
Hi! @Mikelobi , 👋 

Good job so far, 👍 
There are some issues that you still need to work on to go to the next project but you are almost there.  These changes add some awesomeness to your awesome project and code skills

**To Highlight** ✔️ 
---
Meaningful commit messages ✅ 
Highly organized and clean code ✅ 

**Required Changes** ♻️ 
---
- There is no Github action for your workflow 😮, please fix this.
- Please check the comments under the review.


Cheers and Happy coding!👏👏

**Review by keroloussamy — APPROVED** [review]
Hi @Mikelobi, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To highlight 🎯

- Comments are polite.✔️
- Includes an explanation. ✔️
- Comments are clear and easy to understand. ✔️

clap



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**